packages/cozy-harvest-lib/src/components/hooks/useMaintenanceStatus.jsx

```diff
@@ -16,7 +17,11 @@ const useMaintenanceStatus = (client, slug) => {
       const appStatus = await registry.fetchApp(slug)
       const isInMaintenance = get(appStatus, 'maintenance_activated', false)
       const messages = get(appStatus, 'maintenance_options.messages', {})
-      setMaintenanceStatus({ isInMaintenance, messages })
+      setMaintenanceStatus({
+        isInMaintenance,
+        isMaintenanceLoaded: true,
+        messages
+      })
```

Comment:
Nit: I wonder if it would make sense to have the same kind of API as the query result.

```
{
  data: { isInMaintenance, messages },
  lastUpdate,
  lastFetch,
  fetchStatus
}
```